File: tests/openmp_examples/data_environment/udr.2.c
#pragma omp declare reduction(min : struct point) combiner( omp_out.x = omp_in.x > omp_out.x ? omp_out.x : omp_in.x, omp_out.y = omp_in.y > omp_out.y ? omp_out.y : omp_in.y ) initializer( omp_priv = { 2147483647, 2147483647 } )
#pragma omp declare reduction(max : struct point) combiner( omp_out.x = omp_in.x < omp_out.x ? omp_out.x : omp_in.x, omp_out.y = omp_in.y < omp_out.y ? omp_out.y : omp_in.y ) initializer( omp_priv = { 0, 0 } )
#pragma omp parallel for reduction(min:minp) reduction(max:maxp)

File: tests/openmp_examples/data_environment/udr.2.f90
!$omp   declare reduction( min : point ) combiner(omp_out = point(min(omp_out%x, omp_in%x), min(omp_out%y, omp_in%y))) initializer(omp_priv = point(HUGE(0), HUGE(0)))
!$omp   declare reduction( max : point ) combiner(omp_out = point(max(omp_out%x, omp_in%x), max(omp_out%y, omp_in%y))) initializer(omp_priv = point(0, 0))
!$omp   parallel do reduction(min: minp) reduction(max: maxp)
